### PR TITLE
Add Applications, 2 sub-tasks here

### PR DESCRIPTION
**Pull request by Colin1841**
2 Subtasks included here:
Copied Custom Applications: https://jira.d2iq.com/browse/D2IQ-84556
Added scope field: https://jira.d2iq.com/browse/D2IQ-84557

### CODE REVIEW

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ You can define the following fields:
 | category    | general              | 1 or more categories for this application. Categories are used to group applications in the UI. |
 | overview    |                      | Markdown overview used on the application detail page in the UI.                                |
 | icon        |                      | Base64 encoded icon SVG file used for application logos in the UI.                              |
+| scope       |                      |                                                                                                 |
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md:19]:
```suggestion
| scope       | workspace            | List of scopes, can be workspace and/or project currently.                                                                                               |
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+layout: layout.pug
+navigationTitle: Create a Git Repository
+title: Create a Git Repository
+menuWeight: 10
+excerpt: Create a Git Repository in the Project namespace
```
Comment (gracedo) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md:6]:
```suggestion
excerpt: Create a Git Repository in the Workspace namespace
```

Please adapt all these pages to Workspace context

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+layout: layout.pug
+navigationTitle: Deploy a Custom Application from the Project Catalog
+title: Deploy a Custom Application from the Project Catalog
+menuWeight: 50
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md:5]: beta: true

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+---
+layout: layout.pug
+navigationTitle: Git repository structure
+title: Git repository structure
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md:4]: beta: true

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+layout: layout.pug
+navigationTitle: Custom applications
+title: Custom applications
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/index.md:4]: beta: true

**Review by cneth — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md (hunk 2 above)
Comment (gracedo) [reply pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md:7]: ^^ bump on this - these pages were copied over from projects/, need to update all the project references to Workspace context

**Review by gracedo — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md (hunk 3 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md:5]:
```suggestion
menuWeight: 50
beta: true
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md (hunk 4 above)
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md:4]:
```suggestion
title: Git repository structure
beta: true
```

**Comment by Colin1841** [comment]
@gracedo - I applied a global term change, projects > workspace, and maintained the cases. However, I have a feeling this might have been a shotgun when a fly swatter was needed approach, so let me know if I need to back these out and do this manually!

**Review by gracedo — APPROVED** [review]
I don't think these pages should be labeled beta since these "features" are already out in 2.1.1

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 6]
```diff
@@ -2,10 +2,11 @@
 layout: layout.pug
 navigationTitle: Application Metadata
 title: Application Metadata
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md:5]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle: Application Metadata
+title: Application Metadata
+beta: true 
+menuWeight: 40
+excerpt: To display more information about custom applications in the UI, define a metadata.yaml file for each application in the git repository.
+---
+You can define how custom applications are displayed in the Kommander UI by defining a `metadata.yaml` file for each application in the git repository. You must define this file at `services/<application>/metadata.yaml` for it to process correctly.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md:9]:
```suggestion
You can define how custom applications display in the Kommander UI by defining a `metadata.yaml` file for each application in the git repository. You must define this file at `services/<application>/metadata.yaml` for it to process correctly.
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+| Field       | Default              | Description                                                                                     |
+| ----------- | -------------------- | ----------------------------------------------------------------------------------------------- |
+| displayName | falls back to App ID | Display name of the application for the UI.                                                     |
+| description | ""                   | Short description, should be a sentence or two, displayed in the UI on the application card.    |
+| category    | general              | 1 or more categories for this application. Categories are used to group applications in the UI. |
+| overview    |                      | Markdown overview used on the application detail page in the UI.                                |
+| icon        |                      | Base64 encoded icon SVG file used for application logos in the UI.                              |
+
+None of these fields are required for the application to be displayed in the UI.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md:21]:
```suggestion
None of these fields are required for the application to display in the UI.
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
+---
+layout: layout.pug
+navigationTitle: Create a Git Repository
+title: Create a Git Repository
+menuWeight: 10
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md:6]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle: Application Metadata
+title: Application Metadata
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/application-metadata/index.md:5]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
+---
+layout: layout.pug
+navigationTitle: Deploy a Custom Application from the Workspace Catalog
+title: Deploy a Custom Application from the Workspace Catalog
+menuWeight: 50
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md:6]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/git-repo-structure/index.md:5]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/index.md [hunk 13]
```diff
@@ -0,0 +1,10 @@
...
+menuWeight: 30
+excerpt: Custom applications are third-party applications you have added to the Kommander Catalog.
+beta: true
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/index.md:7]:
```suggestion
beta: false
```

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (cmastr) [reply pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/application-metadata/index.md]: This is the 2.2 docs. DId we add these feature descriptions to 2.1 as well? @Colin1841 @gracedo

**Comment by Colin1841** [comment]
Duplicated Workspace Custom application pages to 2.1: 2.1/workspaces/applications/catalog-applications/custom-applications/

**Pull request merged**